src/client/components/general/IngredientCard/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './ingredientCard.scss'
+
+function index({ name, img, description, index }) {
+
+  return (
+    <article className='ingredient' key={index}>
+        <img src={img} alt='ingredient'></img>
```

Comment:
alt could be a prop as well, to better prepare this component for future uses in other places of the application